### PR TITLE
Regenerate go.mod and go.sum

### PR DESCRIPTION
**Pull request by mkrump**
## Description

The [go dependency update was failing](https://circleci.com/gh/transcom/mymove/183396) b/c of an indirect dependency that had not been updated to work with Go 1.13 

```
# github.com/gopherjs/gopherjs/compiler
../../go/pkg/mod/github.com/gopherjs/gopherjs@v0.0.0-20190915194858-d3ddacdb130f/compiler/compiler.go:20:9: undefined: ___GOPHERJS_REQUIRES_GO_VERSION_1_12___
exit status 1
Makefile:290: recipe for target 'go_deps_update' failed
```

It didn't appear that this module was being used, so we decided to regenerate the `go.mod` and `go.sum` files and indeed problem indirect dependency was removed. Also, the `gin` dependency was failing to download b/c of a package rename, so a replace was added to the `go.mod` correct for this.

## Reviewer Notes

Double check the indirect dependencies that are being removed. It gives me pause that running `go mod tidy` doesn't result in any change and is not doing the removal for us as I thought that was what [it was supposed to do](https://tip.golang.org/cmd/go/#hdr-Maintaining_module_requirements])?

## Setup

```sh
make clean
rm go.mod go.sum
go mod init github.com/transcom/mymove
go mod tidy

## Add the replace statments to go.mod
// transcom/sqlx v1.2.1 is just jmoiron's 1.2.0 with custom driver fixes
// This is a temporary solution till https://github.com/jmoiron/sqlx/pull/560
// is merged or a better solution is completed as mentioned in
// https://github.com/jmoiron/sqlx/pull/520
replace github.com/jmoiron/sqlx v1.2.0 => github.com/transcom/sqlx v1.2.1
// https://github.com/codegangsta/gin/issues/154#issuecomment-544391671
// This fixes an issue that was being caused due to urfave/cli v1.21.0
// being renamed.
replace gopkg.in/urfave/cli.v1 => github.com/urfave/cli v1.21.0

make build
make server_test
```

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/n/projects/2181745/stories/169018664) for this change

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -97,3 +84,8 @@ require (
 // is merged or a better solution is completed as mentioned in
 // https://github.com/jmoiron/sqlx/pull/520
 replace github.com/jmoiron/sqlx v1.2.0 => github.com/transcom/sqlx v1.2.1
+
+// https://github.com/codegangsta/gin/issues/154#issuecomment-544391671
+// This fixes an issue that was being caused due to urfave/cli v1.21.0
+// being renamed.
+replace gopkg.in/urfave/cli.v1 => github.com/urfave/cli v1.21.0
```
Comment (reggieriser) [inline go.mod]: If we add this `replace`, does that mean we can remove it from the `excludes` here?  Or is that still relevant?

https://github.com/transcom/mymove/blob/bf8f1bed2b9e1bb720382a534de62384047b085a/cmd/update_deps/main.go#L22-L24

Comment (mkrump) [reply go.mod]: Seems like yes, but what's the best way to test out?

Comment (reggieriser) [reply go.mod]: I haven't tried this lately, but following the `Setup` section in #2501 (where I first introduced the `excludes`) may still work.

Comment (mkrump) [reply go.mod]: That still worked. Passed locally w/o the ignore, so added in 041eeee15.

**Pull request merged**